packages/cli/subtasks/pick-function.js

```diff
@@ -0,0 +1,32 @@
+const { subtask } = require('hardhat/config');
+const constants = require('../constants');
+const { SUBTASK_PICK_FUNCTION } = require('../task-names');
+const inquirer = require('inquirer');
+
+subtask(SUBTASK_PICK_FUNCTION, 'Pick a function from the given contract').setAction(
+  async (taskArguments, hre) => {
+    const abi = hre.deployer.deployment.abis[hre.cli.contractName];
+    const abiFunctions = abi.filter((abiItem) => abiItem.name && abiItem.type === 'function');
+
+    abiFunctions.splice(0, 0, { name: constants.BACK });
```

Comment:
Does this `.splice` work correctly? Being the value to remove `{ name: constants.BACK }` I think that it will compare 2 different instances of the same object and not going to remove it.

May, you can do this directly:
```suggestion
    const abiFunctions = abi
      .filter((abiItem) => abiItem.name && abiItem.type === 'function');
      .filter((abiItem) => abiItem.name !== constants.BACK);
```

Reply:
Delete count is zero, I'm actually trying to inject the element, not remove it.